react/tic-tac-toe/src/app/screens/Players/index.js

```diff
@@ -0,0 +1,13 @@
+import React, { Fragment } from 'react';
+
+import TopBar from '../Game/components/TopBar';
+
+function Players() {
+  return (
+    <Fragment>
+      <TopBar />
+    </Fragment>
```

Comment:
I get the feeling you are going to add more things here, aren't you ? 👀

Reply:
yes! in the next card